Autopilot/Path Manager/PathManager.c

```diff
@@ -667,6 +674,19 @@ void copyGPSData(){
     pmData.checkbyteDMA2 = generatePMDataDMAChecksum2();
 }
 
+static void checkForFirstGPSLock(){
+	static char gpsLockFlag = 1;
```

Comment:
So i actually looked it up and according to http://stackoverflow.com/questions/5033627/static-variable-inside-of-a-function-in-c it should still retain it's value even if declared inside the function since the initialization is done at compile time only. I also then tried it with a small piece of test code to confirm and it does retain the value. I can talk to you more about this in the bay today